File: src/main/java/dev/fynf/randomfood2/entities/FoodRepository.java
package dev.fynf.randomfood2.entities;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface FoodRepository extends CrudRepository<Food, Long> {
  List<Food> findAllByIdNotIn(List<Long> ids);

}

File: src/main/kotlin/dev/fynf/randomfood2/entities/FoodRepository.kt
package dev.fynf.randomfood2.entities

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface FoodRepository : CrudRepository<Food?, Long?> {
    fun findAllByIdNotIn(ids: List<Long?>?): List<Food?>?
}